Content.Server/Chat/Commands/NoticeCommand.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.Chat.Managers;
+using Content.Shared.Administration;
+using Content.Shared.Chat;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Enums;
+
+namespace Content.Server.Chat.Commands;
+
+[AnyCommand]
+internal sealed class NoticeCommand : IConsoleCommand
+{
+    [Dependency] private readonly IChatManager _chatManager = default!;
+
+    public string Command => "notice";
+    public string Description => Loc.GetString("notice-command-description");
+    public string Help => Loc.GetString("notice-command-help", ("command", Command));
+
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (shell.Player is not IPlayerSession player)
+        {
+            shell.WriteError("This command cannot be run from the server.");
+            return;
+        }
+
+        if (player.Status != SessionStatus.InGame)
+            return;
+
+        if (player.AttachedEntity is not { })
+        {
+            shell.WriteError("You don't have an entity!");
+            return;
+        }
```

Comment:
Still needs localising please, everything the client can see needs to be localised.